deploy-satellite-update-playbook.yml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Also as per the doc convention:
update - means an update a new minor version
upgrade - means an upgrade to a new major version.
I am using the same convention here - so satellite-update-playbook.yml should be okay, what do you think?